conda-store-server/alembic/versions/5ad723de2abd_adding_container_registry_value_to_enum.py

```diff
@@ -0,0 +1,24 @@
+"""Adding CONTAINER_REGISTRY value to enum
+
+Revision ID: 5ad723de2abd
+Revises: 8d63a091aff8
+Create Date: 2022-08-05 22:14:34.110642
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '5ad723de2abd'
+down_revision = '8d63a091aff8'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
@pierrotsmnrd this has to do with my limited knowledge of sqlalchemy. Is there a good way to modify this enum by adding a new value? Downgrade also sounds tricky since I'd have to drop all build artifacts which are of that enum type.

Reply:
Also this query is postgresql dependant and I'll need to make it cross database.

Is a better approach to rename the column, create a new enum, then delete the old one?